src/openklant2/client.py

```diff
@@ -18,3 +32,37 @@ def __init__(self, token: str, api_root: str):
         self.partij = PartijResource(self.http_client)
         self.partij_identificator = PartijIdentificatorResource(self.http_client)
         self.digitaal_adres = DigitaalAdresResource(self.http_client)
+        self.klant_contact = KlantContactResource(self.http_client)
+        self.onderwerp_object = OnderwerpObjectResource(self.http_client)
+        self.actor = ActorResource(self.http_client)
+        self.interne_taak = InterneTaakResource(self.http_client)
+        self.betrokkene = BetrokkeneResource(self.http_client)
+
+    def paginator(
```

Comment:
This is the same code as used for the resources. Perhaps make a `PaginationMixin` out of it if we need it in multiple places.